src/components/TreesMap/DeckGLMap.tsx

```diff
@@ -558,7 +558,9 @@ class DeckGLMap extends React.Component<DeckGLPropType, DeckGLStateType> {
             reuseMaps
             mapStyle='mapbox://styles/technologiestiftung/ckke3kyr00w5w17mytksdr3ro'
             preventStyleDiffing={true}
-            mapboxApiAccessToken={process.env.MAPBOX_API_KEY}
+            mapboxApiAccessToken={
+              'pk.eyJ1IjoibHVjYXNvZXRoIiwiYSI6ImNrbWFyYWRydDF1OTkyeW42MXVmeWxtOG4ifQ.mD2nM5vbWilfbo61ZHc--Q'
```

Comment:
You should never expose API tokens. Now, this is public. This is obviously an accident, but you should be careful. We should reset the token but I'm not sure if this would have an effect on the live site. @ff6347 do you know what are the consequences here?

Reply:
its my own api key, I have disabled it